scaletest/dashboard/chromedp.go

```diff
@@ -178,6 +192,13 @@ func initChromeDPCtx(ctx context.Context, log slog.Logger, u *url.URL, sessionTo
 		}
 	}
 
+	// force a viewport size of 1024x768 so we don't go into mobile mode
+	if err := chromedp.Run(cdpCtx, chromedp.EmulateViewport(1024, 768)); err != nil {
```

Comment:
Glad you found this! I totally forgot about the default screen size of headless. 😮‍💨